### PR TITLE
Improvements to tests

### PR DESCRIPTION
**Pull request by NissimHadar**
# Description
GitHub info is now parameterised to simplify testing.
Fixed avatar used for tests.
Disabled TAA where it is causing tests to fail.

### CODE REVIEW

File: tests/utils/autoTester.js [hunk 1]
```diff
@@ -161,6 +174,24 @@ module.exports.setupTest = function (primaryCamera) {
...
+    MyAvatar.skeletonModelURL = "https://highfidelity.com/api/v1/commerce/entity_edition/813addb9-b985-49c8-9912-36fdbb57e04a.fst?certificate_id=MEUCIQDgYR2%2BOrCh5HXeHCm%2BkR0a2JniEO%2BY4y9tbApxCAPo4wIgXZEQdI4cQc%2FstAcr9tFT9k4k%2Fbuj3ufB1aB4W0tjIJc%3D";
+
+    // Wait for skeleton to load (for now - only in test mode)
+    if (typeof Test !== 'undefined') {
+        Test.waitIdle();
+    }
+
+    // Set Avatar to T-pose
+    for (var i = 0; i < MyAvatar.getJointNames().length; ++i) {
+       MyAvatar.setJointData(i, MyAvatar.getDefaultJointRotation(i), MyAvatar.getDefaultJointTranslation(i));
```
Comment (AndrewMeadows) [inline tests/utils/autoTester.js]: Bad indentation at line 193.

File: tests/utils/autoTester.js [hunk 2]
```diff
@@ -212,7 +243,7 @@ module.exports.setupTest = function (primaryCamera) {
     spectatorCameraConfig.resetSizeSpectatorCamera(1920, 1080);
     spectatorCameraConfig.vFoV = 45;
     Render.getConfig("SecondaryCameraJob.ToneMapping").curve = 0;
-	  Render.getConfig("SecondaryCameraJob.DrawHighlight").enabled = false;
+	Render.getConfig("SecondaryCameraJob.DrawHighlight").enabled = false;
```
Comment (AndrewMeadows) [inline tests/utils/autoTester.js]: Bad indentation at line 246.  Two spaces were removed but only one should have been.

File: tests/engine/testRecursive.js [hunk 3]
```diff
@@ -1,43 +1,47 @@
-// This is an automatically generated file, created by auto-tester on May 16 2018, 9:09
+// This is an automatically generated file, created by auto-tester on May 24 2018, 13:34
+
+user = "highfidelity/"
+repository = "hifi_tests/"
+branch = "master/"
```
Comment (AndrewMeadows) [inline tests/engine/testRecursive.js]: The semicolons at the end of JS lines are optional, right?  But they are preferred?

File: tests/engine/render/textures/testRecursive.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// This is an automatically generated file, created by auto-tester on May 24 2018, 13:34
+
+user = "highfidelity/"
+repository = "hifi_tests/"
+branch = "master/"
```
Comment (AndrewMeadows) [inline tests/engine/render/textures/testRecursive.js]: The semicolons at the end of JS lines are optional, right? But they are preferred?

File: tests/engine/render/testRecursive.js [hunk 5]
```diff
@@ -1,35 +1,40 @@
-// This is an automatically generated file, created by auto-tester on May 16 2018, 9:09
+// This is an automatically generated file, created by auto-tester on May 24 2018, 13:34
+
+user = "highfidelity/"
+repository = "hifi_tests/"
+branch = "master/"
```
Comment (AndrewMeadows) [inline tests/engine/render/testRecursive.js]: The semicolons at the end of JS lines are optional, right? But they are preferred?

File: tests/engine/render/shadows/testRecursive.js [hunk 6]
```diff
@@ -1,12 +1,16 @@
-// This is an automatically generated file, created by auto-tester on May 16 2018, 9:09
+// This is an automatically generated file, created by auto-tester on May 24 2018, 13:34
+
+user = "highfidelity/"
+repository = "hifi_tests/"
+branch = "master/"
```
Comment (AndrewMeadows) [inline tests/engine/render/shadows/testRecursive.js]: The semicolons at the end of JS lines are optional, right? But they are preferred?

File: tests/engine/render/testRecursive.js (hunk 5 above)
Comment (NissimHadar) [reply tests/engine/render/testRecursive.js]: That is a good question.  I personally do prefer them.

**Pull request merged**